clients/src/main/java/org/apache/kafka/common/config/provider/FileConfigProvider.java

```diff
@@ -93,9 +129,16 @@ public ConfigData get(String path, Set<String> keys) {
         }
     }
 
-    // visible for testing
-    protected Reader reader(String path) throws IOException {
-        return Files.newBufferedReader(Paths.get(path));
+    private String toString(Path path) {
```

Comment:
I think in general we shouldn't call this `toString`. People would confuse it for the actual `toString` method and would suppose the same or similar behavior. For instance `toString` methods aren't expected to throw exceptions.
So in my opinion we should call this something like `readAll` or `readAllBytes`.